src/Visualizer/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
there is same file in root folder. why do we this in this folder too?

Reply:
If there was no .eslintrc.json in the Visualizer folder, an error occurred during `npm run lint`, so I put it inside by referring to [here](https://eslint.org/docs/user-guide/configuring/configuration-files#relative-glob-patterns).

Reply:
Thanks for the link. As I understand by short reading, your link shows for two separate program: app and sever.
But for our project, it's one application with several modules.
Can you share your program with `npm run lint` ?

Reply:
[here](https://github.com/wnsdud4197/ONE-vscode/runs/3611431352?check_suite_focus=true) is the error, 
We used react to create visualization tools. So we needed a module for react, and in that folder, only modules that run react and build exist.

Reply:
> ESLint couldn't find the config "react-app" to extend from. Please check that the name of the config is correct.
> home/runner/work/ONE-vscode/ONE-vscode/src/Visualizer/package.json

I don't know about detail (I'm newbie in npm, vscode-ext) but my guess is that you need to update `package.json` file in the root folder.

What I've found: https://stackoverflow.com/questions/60431931/eslint-error-eslint-couldnt-find-the-config-react-app

Reply:
We tried it. But I faced a different [problem](https://github.com/wnsdud4197/ONE-vscode/runs/3616973147?check_suite_focus=true).
That's why I added it by referring to eslint. 😥

Reply:
By adding it, the problem of 'npm run lint' was solved, but the problem occurred in 'bash infra/format'.

Reply:
You can see format error 
(1) click at details
![image](https://user-images.githubusercontent.com/4616940/133566437-adcbbb48-0835-4209-ae51-bdccee9867b7.png)
(2) download format.patch zip file
![image](https://user-images.githubusercontent.com/4616940/133566366-0a47ada9-f756-4334-b085-195fcc034c61.png)